### DISCUSSION TITLE
PMF function is not correctly looking through my Cantera flame data

### DISCUSSION THREAD

**Discussion by Nadexterbrown** [AMReX-Combustion/PeleC#712] [Q&A] [answered]
Hello,
I am attempting to setup a personal planar flame model. However, currently I am encountering a problem with the PMF function in prob.H only looking at the first and second "data points" (temperature, velocity, etc.). This is resulting in an incorrect initial conditions with nothing being implemented into the domain. I have setup the simulation using a hybrid code between the PMF and Converging Nozzle example cases. I am unsure as to why the pmf function is behaving in this way, as it's just a copy paste from the PMF case, which is also almost identical to the pmf function provided in the Soot Flame case. This problem is occurring during the initialization process prior to BC being defined to my current understanding. 

The inputs into the pmf function within prob.H are at the correct grid location, but the read flame.dat file information appears to be incorrect. 

Any help would be extremely appreciated. I have provided some blocks from the initialization process showing the problem I am encountering. If anything else is needed to help convey my problem please let me know and I will happily provide. 

Nolan

System Configuration
MacBook Air M2
8 GB Memory
Sonoma 14.0

-------------------------------------
Cantera Flame Simulation Results (Stoichiometric H2/O2 flame at 1.0 atm)
Reactant Temperature = 300
Product Temperature = 3073 K 

PeleC Simulation Gridding Parameters
2D (x y)
1e-3 64e-3 [meters]

-------------------------------------
PMF Print Statements to Terminal

0.036125 PMF xlo
0.03625 PMF xhi
0 PMF Interp Low
1 PMF Interp High
0 PMF x1
0.6933333333 PMF x2
300 PMF y1
300 PMF y2
300 PMF y_vector (Temperature)

0.049125 PMF xlo
0.04925 PMF xhi
0 PMF Interp Low
1 PMF Interp High
0   PMF x1
0.6933333333   PMF x2
300   PMF y1
300   PMF y2
300   PMF y_vector ( Temperature)

**Comment by drummerdoc** [comment]
Any idea what the issue was?

**Reply by Nadexterbrown** [reply, marked as answer]
The issue stemmed from the fact that PeleC is written using cgs units and not meters, kilogram, seconds.  This let to the code trying to initialize the flame on a domain that was too small, which it couldn't do.
